pyproject.toml

```diff
@@ -45,7 +45,7 @@ before-build = [
 ]
 
 [tool.cibuildwheel.macos.environment]
-LDFLAGS="-Wl,-rpath,$CONDA/envs/build/lib -L$CONDA/envs/build/lib"
+LDFLAGS="-Wl,-rpath,$CONDA/envs/build/lib -L$CONDA/envs/build/lib -lomp -ljemalloclocal"
```

Comment:
(How) is this related to https://github.com/Quantco/tabmat/blob/fbf7755aa48ada1d72dec5d61b7fd9d9f23fdcc6/setup.py#L62-L86 ?

Reply:
With the fix above, it might work without these flags again. Let's try.